prompt-note/task/index.js

```diff
@@ -0,0 +1,84 @@
+const prompt = require("prompt-sync")();
+
+let count = 0;
+let listItem = [];
+const resultMessage = () => console.log("종료 되었습니다.");
+
+// 질문
+const question = () => {
+  const message = prompt(
+    "1. 전체메모 조회 / 2. 메모 입력 / 3. 메모 모두 삭제하기 / 4. 메모 수정하기 / 0. 종료 : "
+  );
+  const feedback = +message;
+
+  switch (feedback) {
+    case 1:
+      list();
+      break;
+    case 2:
+      write();
+      break;
+    case 3:
+      remove();
+      break;
+    case 4:
+      update();
+      break;
+  }
+};
+
+// 뒤로가기
+const back = () => {
+  const message = prompt("0. 뒤로가기 / 1: 종료 : ");
+  const feedback = Number(message);
+
+  if (feedback === 0) {
+    question();
+  }
+};
+
+// 전체메모 조회
+const list = () => {
+  if (!listItem.length) {
+    console.log("저장된 메모가 없습니다.");
+  } else {
+    listItem.map((item, index) => {
+      console.log(`${index + 1}. ${item}`);
+    });
+  }
+
+  back();
+};
+
+// 메모 입력
+const write = () => {
+  if (listItem.length >= 5) {
+    const text = console.log(
+      "더이상 메모를 저장할 수 없습니다. (최대 5개 저장 가능)"
+    );
+  } else {
+    const text = prompt("메모를 입력해주세요 : ");
+    listItem.push(text);
+  }
+
+  back();
+};
+
+// 메모 전체 삭제
+const remove = () => {
+  if (listItem.length === 0) {
+    const text = console.log("저장된 메모가 없습니다.");
```

Comment:
요기도 변수에 할당 안하고 그냥 적어도 괜찮습니당~
```suggestion
console.log("저장된 메모가 없습니다.");
```